src/compliance/NoComplianceCompetition.sol

```diff
@@ -0,0 +1,24 @@
+pragma solidity ^0.4.19;
+
+import "./NoCompliance.sol";
+
+/// @title Compliance Contract
+/// @author Melonport AG <team@melonport.com>
+/// @notice Only allow manager to invest in the Fund.
```

Comment:
outdated comment, would be "Anyone is allowed to invest in the Fund". 

What is the difference between NoComplianceCompetition and NoCompliance ?

Reply:
Changed the comment. It's basically the same with the added isCompetitionAllowed function that always returns true (actual version checks for whitelist and competition is active conditions)

Reply:
gotcha